src/backend/tests/test_crawler_expunger.py

```diff
@@ -254,3 +257,72 @@ def test_expunger_for_record_with_mj_over_21(record_with_mj_over_21):
             status=EligibilityStatus.ELIGIBLE, reason="Eligible now", date_will_be_eligible=date(2001, 3, 3)
         )
     }
+
+@pytest.fixture
+def record_with_mj_under_21_and_traffic_violation():
+    return CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_TRAFFIC_CONVICTION,})
+
+def test_expunger_for_record_with_mj_under_21_not_blocked_by_traffic(record_with_mj_under_21_and_traffic_violation):
+    expunger_result = Expunger.run(record_with_mj_under_21_and_traffic_violation)
+
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[0].charge_type, MarijuanaUnder21)
+    assert isinstance(record_with_mj_under_21_and_traffic_violation.charges[1].charge_type, TrafficViolation)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1999, 3, 3)) #one year after conviction
+    assert expunger_result["CASEJD1-2"] == TimeEligibility(status=EligibilityStatus.INELIGIBLE,
+                                                           reason="Never. Type ineligible charges are always time ineligible.",
+                                                           date_will_be_eligible=date.max())
+
+def test_expunger_for_record_with_mj_under_21_with_non_traffic_charge():
+    record = CrawlerFactory.create(
+        record=YoungDoe.SINGLE_CASE_RECORD,
+        cases={"CASEJD1": CaseDetails.CASE_MJ_AND_CIVIL_CONVICTION})
+
+    expunger_result = Expunger.run(record)
+
+    assert isinstance(record.charges[1].charge_type, CivilOffense)
+
+    assert expunger_result["CASEJD1-1"] == TimeEligibility(status=EligibilityStatus.ELIGIBLE,
+                                                           reason="Eligible now",
+                                                           date_will_be_eligible=date(1998, 3, 3)+relativedelta(years=1))
```

Comment:
This is interesting. I would expect +3 years just like the test below.

After more digging, it looks like the CivilOffense is being picked up as a dismissal instead of a conviction. Or more specifically, since the parser doesn't recognize the
```
2. Civil Offense
Convicted
```
that you wrote it in the fixture, by omission, it becomes dismissed. In any case this tests the same thing as `test_expunger_for_record_with_mj_under_21_with_non_traffic_charge_not_subject_to_137226` so I would just delete this test (while keeping the naming).

Reply:
And if you're curious as to why it doesn't pick up, it looks like you missed a semicolon after the `&nbsp`. As you can see, the case parser is fragile but we've received assurances that we can rely on the HTML structure not changing, so I am not concerned.